webpack/scenes/Subscriptions/SubscriptionActions.js

```diff
@@ -106,7 +106,7 @@ export const resetTasks = () => (dispatch) => {
 
 export const handleTask = task => async (dispatch, getState) => {
   if (selectIsPollingTask(getState(), SUBSCRIPTIONS)) {
-    if (!task.pending) {
+    if (!task.pending && task.result !== 'pending') {
```

Comment:
I wonder if this should be in a selector in TaskSelectors

Reply:
`selectTaskIsDefinitelyNotPendingOrAtLeastWerePrettySure`

Reply:
Added the selector.   Since we're inside a thunk as opposed to a React component, I had to use `getState()` so it feels kind of clunky to me.  I used a separate commit just in case.. let me know your thoughts